ci_framework/roles/dlrn_report/tasks/dlrn_report_results.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Set zuul-log-path fact
+  ansible.builtin.include_role:
+    name: set-zuul-log-path-fact
+
+- name: Get hash related data from repo_setup role
+  ansible.builtin.include_role:
+    name: repo_setup
+    tasks_from: artifacts.yml
+
+- name: Report results to dlrn for the tested hash
+  ansible.builtin.shell:
+    cmd: >-
+      {% if zuul_success|bool %}
```

Comment:
where is zuul_success set i cannot quickly find that eg there [1] [2]

EDIT: ok it is a zuul thing, found it at [3]


[1] https://codesearch.rdoproject.org/codesearch/?q=zuul_success

[2] https://github.com/search?q=org%3Aopenstack-k8s-operators%20zuul_success&type=code

[3] https://zuul-ci.org/docs/zuul/latest/job-content.html#var-zuul_success

Reply:
"what if" that param isn't set? Maybe ensuring it's set and then checking its value.. ?

Reply:
Done